### ISSUE TITLE
Lacking consistency on determining what are manifests

### ISSUE THREAD

**Issue by SeaDve** [bilelmoussaoui/flatpak-vscode#117]
On `package.json`, it is required to have at least three elements for it to allow `jsonValidation`/`yamlValidation`. But in the code, we allow at least two elements.

**Comment by bilelmoussaoui** [comment]
Indeed, I wonder if we could provide such integration using actual code instead of package.json, this way - we could potentially only make it usable for actual detected manifest
